### ISSUE TITLE
Accessibility Testing for  [My Education Benefits, VA Education Inbox]

### ISSUE THREAD

**Issue by cesaresantiago** [department-of-veterans-affairs/va.gov-team#50417]
### Product information

- [X] Team name, product name, and feature name have been added to the title of this issue.
- [X] Team label, product label and feature label (if applicable) have been added to this issue.

### Who completed the use of color and color contrast test?

Bbartlett26

### Use of color and color contrast checks

- [X] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
- [X] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
- [X] Non-text elements have a 3:1  contrast ratio to their background and to neighboring elements (or better)
- [X] Color is not the only way to distinguish links from other text (eg. links are underlined)
- [X] Any charts, maps, infographics, and tables convey all information without only relying on color
- [X] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")

### How did color testing go?

For color contrast testing we do manual WAVE tests. We also validate color and contrast using AXE in automation tests, validating with a project library that checks against VA 508 rules. No major issues were found.

### Who completed the axe scans?

Bbartlett26

### axe checks

- [X] Each page has been scanned using axe (results shared below)
- [X] axe is integrated into your end-to-end testing

### axe DevTools scan results

Axe results attached.

[axeDt-reports_educationLetters.zip](https://github.com/department-of-veterans-affairs/va.gov-team/files/10169671/axeDt-reports_educationLetters.zip)

### How did axe testing go?

Axe results were addressed by developers. Last report (attached) found 0 violations.

### Who completed the content zoom and reflow test?

Bbartlett26

### Content zoom and reflow checks

- [X] All page elements are readable and usable at 200% zoom
- [ ] All page elements are readable and usable at 300% zoom
- [ ] All page elements are readable and usable at 400% zoom

### How did content zoom and reflow testing go?

We test up to 200% per practice for all DGI apps for VA. 

However we are reusing [VA.gov](http://va.gov/) platform components with no modifications that have already passed this testing.

### Who completed the keyboard navigation test?

Bbartlett26

### Keyboard navigation checks

- [X] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
- [X] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
- [X] All elements under focus have a visible focus indicator
- [X] The order of [Tab] stops made sense and was appropriate for completing tasks

### How did keyboard testing go?

We check keyboard navigation issues with wave, axe and manual testing. We also do some voiceover testing.

No major issues found.
